core/src/main/java/com/netease/arctic/scan/TableEntriesScan.java

```diff
@@ -147,14 +149,21 @@ public Builder project(Schema schema) {
       return this;
     }
 
+    public Builder withMetadataTableType(MetadataTableType metadataTableType) {
```

Comment:
AFAIK, this class is used to scan table entries table, why should we add this method here?

Reply:
I consider using MetadataTableType this parameter to control the type of file to be scanned.
Otherwise, the original class can only get files of type `MetadataTableType.ENTRIES`.

Reply:
It's also considered to extend the use of `TableEntriesScan`